OpenRA.Mods.Common/Traits/Modifiers/WithColoredOverlay.cs

```diff
@@ -45,10 +51,8 @@ IEnumerable<IRenderable> ModifiedRender(Actor self, WorldRenderer wr, IEnumerabl
 			{
 				yield return a;
 
-				if (!a.IsDecoration)
-					yield return a.WithPalette(wr.Palette(Info.Palette))
-						.WithZOffset(a.ZOffset + 1)
-						.AsDecoration();
+				if (!a.IsDecoration && a is IModifyableRenderable ma)
```

Comment:
Any particular reason to name this `ma` instead of `mr`?

Reply:
we're looping over `for (var a in r)`, so `mr` would be misleading.